src/promptflow-evals/promptflow/evals/synthetic/simulator/templates/simulator_templates.py

```diff
@@ -0,0 +1,133 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from jinja2 import Environment as JinjaEnvironment
+from jinja2 import FileSystemLoader as JinjaFileSystemLoader
+from jinja2 import meta as JinjaMeta
+
+from promptflow.evals.synthetic.simulator import _template_dir as template_dir
+from promptflow.evals.synthetic.simulator.templates._templates import (
+    ALL_TEMPLATES,
+    CH_TEMPLATES_COLLECTION_KEY,
+    CONTEXT_KEY,
+)
+
+
+class Template:
+    def __init__(self, template_name, text, context_key, content_harm=False, template_parameters=None):
+        self.text = text
+        self.context_key = context_key
+        self.template_name = template_name
+        self.content_harm = content_harm
+        self.template_parameters = template_parameters
+
+    def __str__(self):
+        if self.content_harm:
+            return "{{ch_template_placeholder}}"
+        return self.text
+
+    def __to_ch_templates(self):  # pylint: disable=unused-private-member
+        pass
+
+
+class ContentHarmTemplatesUtils:
+    @staticmethod
+    def get_template_category(key):
+        return key.split("/")[0]
+
+    @staticmethod
+    def get_template_key(key):
+        filepath = key.rsplit(".json")[0]
+        parts = str(filepath).split("/")
+        filename = ContentHarmTemplatesUtils.json_name_to_md_name(parts[-1])
+        prefix = parts[:-1]
+        prefix.append(filename)
+
+        return "/".join(prefix)
+
+    @staticmethod
+    def json_name_to_md_name(name):
+        result = name.replace("_aml", "")
+
+        return result + ".md"
+
+
+class SimulatorTemplates:
+    def __init__(self, rai_client=None):
```

Comment:
Lets remove this since this is not a public class. This can be initialized in Simulator Template directly instead of Simulator